src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -120,6 +120,26 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
         }
 
+    @RESTController.Resource('GET')
+    @handle_send_command_error('osd')
+    def histogram(self, svc_id):
```

Comment:
Please add Python type hint here. How does the returned data look like? Is it a dict or a list? Don't forget to add the result type `str` in case of an error.